.travis.yml

```diff
@@ -127,6 +130,20 @@ install:
         travis_retry sudo apt-get -qq -y install libegl1-mesa;
       fi;
 
+    # Install IPython
+    # No IPython for 2.6
+    # Install using conda with old version for 2.7
+    # Install using pip with new version for > 2.7
```

Comment:
I don't think that will pull the latest versions of ```ipython``` which we need. IMO pip is the right way to pull packages where ```conda``` is not present